test/writer/blueprint_test.exs

```diff
@@ -228,6 +228,7 @@ defmodule BlueBird.Test.Writer.BlueprintTest do
             %Request{
               method: POST,
               path: "/users/:id/pets",
+              title: "Add pet",
```

Comment:
Maybe it could have used a separate test, but decided against it since it would be much unneeded repetition.